components/date-picker/private/year-picklist.jsx

```diff
@@ -43,7 +43,7 @@ const DatepickerYearSelector = createReactClass({
 		const opts = [];
 
 		for (let year = fromYear; year < toYear; year += 1) {
-			opts.push({ label: `${year}`, value: year, id: opts.length });
+			opts.push({ label: `${year}`, value: year, id: String(opts.length) });
```

Comment:
I just saw this today! I didn't catch it when I updated the Combobox props.

Reply:
yup, cool - no worries :)